frontend/libs/studio-content-library/src/ContentLibrary/LibraryBody/pages/CodeListPage/CodeListsActionsBar/CodeListsActionsBar.tsx

```diff
@@ -1,3 +1,4 @@
+import type { ChangeEvent } from 'react';
 import React from 'react';
 import { Search } from '@digdir/designsystemet-react';
```

Comment:
Denne bør vi inkludere i vårt eget komponentbibliotek før vi tar den i bruk.

Reply:
Den er allerede tatt i bruk. Så kanskje løse det i en egen PR?

Reply:
Ja, lag gjerne en egen PR for det.